src/frontend/src/app.html

```diff
@@ -13,11 +13,11 @@
 		<meta content="{{VITE_OISY_DESCRIPTION}}" property="og:description" />
 		<meta content="website" property="og:type" />
 		<meta content="https://oisy.com" property="og:url" />
-		<meta content="https://oisy.com/images/meta-share.jpg" property="og:image" />
+		<meta content="https://oisy.com/images/meta-share-v1.jpg" property="og:image" />
```

Comment:
The new file is a png

Reply:
my mistake! I exported both but put the wrong one. It will be jpg